docs/headless-cms/03-03-metadata.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Metadata
+type: Details
+---
+
+Apart from content, each markdown document displayed in the Kyma Console requires metadata in a specific format.
+
+## Metadata structure
+
+When you create a document, define its `title` and `type`. Place the metadata at the top of your `.md` file, and separate it with three dashes. The following example illustrates the required structure:
+
+```
+---
+title: {Document title}
+type: {Document type}
+---
+```
+
+- The `title` metadata defines the title of your document. 
+- The `type` metadata groups single documents together. Multiple documents that use the same `type` generate a grouping. For example, if you have multiple tutorials, you can group them below a navigation node called **Tutorials**.
+
+>**NOTE:** If there is only one document of a certain type, remove the `type` metadata, so that the document displays well in the UI.
+
+## Metadata display
+
+In the Docs UI, which is the view displayed once you click on the question mark icon in the top-right corner of the Console, the metadata create the left-side navigation structure. The Docs UI displays the documents grouped under a common `type` in the alphanumeric order as per files name.
```

Comment:
`the metadata create the left-side navigation structure` I don't think that making such a statement is ok knowing that markdown metadata is not the only thing that is included in the navigation, there are also things that matter that are part of cdt

Reply:
would adding the word "help" solve the problem? At some point, I had it like `the metadata help create the left-side navigation structure` but wanted to make it shorter.